README.md

```diff
@@ -1,368 +1,257 @@
-3. Longer term, we hope to put in many conveniences into this gem, in terms of Webpack + Rails integration. We're open to suggestions.
-
-## Links
-1. See https://github.com/shakacode/react-webpack-rails-tutorial/ for how to integrate it!
-2. http://www.railsonmaui.com/blog/2014/10/03/integrating-webpack-and-the-es6-transpiler-into-an-existing-rails-project/
-3. http://forum.shakacode.com
-4. If you're looking for consulting on a project using React and Rails, [email us! contact@shakacode.com](mailto: contact@shakacode.com)? You can first join our slack room for some free advice.
-5. We're looking for great developers that want to work with Rails + React with a distributed, worldwide team, for our own
-products, client work, and open source. [More info here](http://www.shakacode.com/about/index.html#work-with-us).
-
-## How is different than the [react-rails gem](https://github.com/reactjs/react-rails)?
-1. `react_on_rails` depends on [webpack](http://webpack.github.io/). `react-rails` integrates closely with sprockets and
-    helps you integrate JSX and the react code into a Rails project.
-2. Likewise, using Webpack as shown in the [react-webpack-rails-tutorial](https://github.com/justin808/react-webpack-rails-tutorial/)
-   does involve some extra setup. However, we feel that tight and simple integration with the node ecosystem is more than
-   worth any minor setup costs.
-3. `react-rails` depends on `jquery-ujs` for client side rendering. `react_on_rails` has it's own JS code that does not
-   depend on jquery.
-
-## Installation Checklist
-1. Include the gems `react_on_rails` and `therubyracer` like [this](https://github.com/shakacode/react-webpack-rails-tutorial/blob/361f4338ebb39a5d3934b00cb6d6fcf494773000/Gemfile#L42) and run `bundle`. Note, you can sustitute your preferable JavaScript engine.
+## Getting Started
+1. Add the following to your Gemfile and bundle install:
   
   ```ruby
   gem "react_on_rails"
   gem "therubyracer"
   ```
-1. Globally expose React in your webpack config like [this](https://github.com/shakacode/react-webpack-rails-tutorial/blob/537c985dc82faee333d80509343ca32a3965f9dd/client/webpack.client.base.config.js#L31):
-  
-  ```javascript
-    module: {
-      loaders: [
-        // React is necessary for the client rendering:
-        { test: require.resolve('react'), loader: 'expose?React' },
-      
-        // For React 0.14
-        { test: require.resolve('react-dom'), loader: 'expose?ReactDOM' }, // not in the server one
-    ```
-    
-    
-1. Require `react_on_rails` in your `application.js` like  [this](https://github.com/shakacode/react-webpack-rails-tutorial/blob/361f4338ebb39a5d3934b00cb6d6fcf494773000/app/assets/javascripts/application.js#L15). It possibly should come after you require `turbolinks`:
-  
+
+2. Run the generator with a simple "Hello World" example:
+
+  ```bash
+  rails generate react_on_rails:install --hello_world_example
```

Comment:
@justin808 good catch this is stale information, there is no way to *not* generate a hello_world example as was requested. I will update accordingly